mlir/tools/mlir-rewrite/mlir-rewrite.cpp

```diff
@@ -320,25 +320,25 @@ LogicalResult markRanges(RewritePad &rewriteState, raw_ostream &os) {
   for (auto it : rewriteState.getOpDefs()) {
     auto [startOp, endOp] = getOpRange(it);
 
-    rewriteState.insertText(startOp, "《");
-    rewriteState.insertText(endOp, "》");
+    rewriteState.insertText(startOp, " <");
+    rewriteState.insertText(endOp, "> ");
 
     auto nameRange = getOpNameRange(it);
 
     if (isGeneric(it)) {
-      rewriteState.insertText(nameRange.Start, "〖");
-      rewriteState.insertText(nameRange.End, "〗");
+      rewriteState.insertText(nameRange.Start, " [");
+      rewriteState.insertText(nameRange.End, "] ");
     } else {
-      rewriteState.insertText(nameRange.Start, "〔");
-      rewriteState.insertText(nameRange.End, "〕");
+      rewriteState.insertText(nameRange.Start, " [");
```

Comment:
Lets perhaps make this something like " !\[" and "\]! " - previously one could differentiate between generic or not form.

Reply:
Okay, I have already updated it.